pkg/apis/application/v1alpha1/repository_types.go

```diff
@@ -187,12 +190,6 @@ func (repo *Repository) GetHelmCreds() helm.Creds {
 	}
 }
 
-// IsHTTPSURL returns true if supplied URL is HTTPS URL
-func IsHTTPSURL(url string) bool {
```

Comment:
While this got removed in a different PR, I see another dependency has been introduced 

```
func getCAPath(repoURL string) string {
	if git.IsHTTPSURL(repoURL) {
		if parsedURL, err := url.Parse(repoURL); err == nil {
			if caPath, err := cert.GetCertBundlePathForRepository(parsedURL.Host); err == nil {
				return caPath
			} else {
				log.Warnf("Could not get cert bundle path for host '%s'", parsedURL.Host)
			}
		} else {
			// We don't fail if we cannot parse the URL, but log a warning in that
			// case. And we execute the command in a verbatim way.
			log.Warnf("Could not parse repo URL '%s'", repoURL)
		}
	}
	return ""
}
```
I will take care of it in a different PR.